lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/new-p2-identity.yaml

```diff
@@ -197,51 +191,28 @@ states:
       next:
         targetState: POST_DCMAW_SUCCESS_PAGE
       not-found:
-        targetState: MULTIPLE_DOC_F2F_CHECK_PAGE
-        checkIfDisabled:
-          f2f:
-            targetState: MULTIPLE_DOC_CHECK_PAGE
+        targetState: MULTIPLE_DOC_CHECK_PAGE
       access-denied:
-        targetState: MULTIPLE_DOC_F2F_CHECK_PAGE
-        checkIfDisabled:
-          f2f:
-            targetState: MULTIPLE_DOC_CHECK_PAGE
+        targetState: MULTIPLE_DOC_CHECK_PAGE
       temporarily-unavailable:
-        targetState: MULTIPLE_DOC_F2F_CHECK_PAGE
-        checkIfDisabled:
-          f2f:
-            targetState: MULTIPLE_DOC_CHECK_PAGE
+        targetState: MULTIPLE_DOC_CHECK_PAGE
       fail-with-no-ci:
-        targetState: MULTIPLE_DOC_F2F_CHECK_PAGE
-        checkIfDisabled:
-          f2f:
-            targetState: MULTIPLE_DOC_CHECK_PAGE
+        targetState: MULTIPLE_DOC_CHECK_PAGE
 
   MULTIPLE_DOC_CHECK_PAGE:
     response:
       type: page
       pageId: page-multiple-doc-check
-    events:
-      ukPassport:
-        targetState: CRI_UK_PASSPORT_J2
-      drivingLicence:
-        targetState: CRI_DRIVING_LICENCE_J3
-      end:
-        targetJourney: INELIGIBLE
-        targetState: INELIGIBLE_SKIP_MESSAGE
-
-  MULTIPLE_DOC_F2F_CHECK_PAGE:
```

Comment:
We probably want to keep this is at first to protect any in-flight journeys while this deploys out?

Reply:
Good point, I guess someone can be in this state for a while that could straddle a deployment